zilliqa/src/consensus.rs

```diff
@@ -731,21 +769,29 @@ impl Consensus {
             return Ok(None);
         }
 
+        let Ok(committee) = self.state.get_stakers_at_block(&block) else {
+            info!("Skipping vote outside of committee");
+            return Ok(None);
```

Comment:
Similarly here, if we aren't able to get the committee at a block should we perhaps return an `Err()`?